security/oidc-client-mutual-tls/src/test/java/io/quarkus/ts/security/oidcclient/mtls/Pkcs12OidcMtlsIT.java

```diff
@@ -15,14 +15,12 @@
  * Keystore file type is automatically detected in following tests by its extension in quarkus-oidc.
  * Extension declared here is used by tests only.
  */
-@Tag("fips-incompatible")
+@Tag("fips-incompatible") // OpenJDK 11
```

Comment:
Can you only enable it for OpenJDK 17? One example is here https://www.baeldung.com/junit-5-conditional-test-execution (you will need to play with it little bit). So adding tag is complication, either you can introduce custom disabling check that ensures FIPS is disabled or you can refactor how we are working with "fips-incompatible" tags, either way, I'd like to disable it on FIPS if possible